opentimelineio_contrib/adapters/ale.py

```diff
@@ -109,6 +111,45 @@ def _parse_data_line(line, columns, fps):
         ))
 
 
+def _video_format(width, height):
+    """Utility function to map a width and hight to an Avid Project Format"""
+
+    format_map = {
```

Comment:
I meant that this map could be moved outside the function as a module level constant -- `AVID_PROJECT_HEIGHT_TO_WIDTH_MAP = {...}` or something.  We're trying to call out these kinds of constant definitions to make them easier to find in the future.

Reply:
ok, I _think_ I understand the request, however I don't think that dict would be very useful unless you are applying the width check for resolutions wider than 1920, which avid (currently) considers CUSTOM

 I can move the _video_format(...) function to the top of the file and make give that a "public" syntax if you think that would be helpful

Reply:
@ssteinbach - I moved and renamed it.  please let me know if that is more what you are thinking

Reply:
Yup, looks great.  Thanks!